database/mssql/scripts/dbscripts/PSP_PIMS_S24_00/Alter Up/25_PIMS_CONTACT_MGR_VW_Fix.SQL

```diff
@@ -0,0 +1,90 @@
+/* Drop and recreate the PIMS_CONTACT_MGR_VW view to include the subselect in 
```

Comment:
This file seems the same in the alter up and the alter down, may not be necessary?